lib/Controller/RoomController.php

```diff
@@ -210,6 +211,33 @@ public function getRooms(int $noStatusUpdate = 0): DataResponse {
 		return new DataResponse($return, Http::STATUS_OK, $this->getTalkHashHeader());
 	}
 
+	/**
+	 * Get listed rooms with optional search term
+	 *
+	 * @NoAdminRequired
+	 *
+	 * @param string $searchTerm search term
+	 * @return DataResponse
+	 */
+	public function getListedRooms(string $searchTerm = ''): DataResponse {
+		$event = new UserEvent($this->userId);
+		$this->dispatcher->dispatch(self::EVENT_BEFORE_LISTED_ROOMS_GET, $event);
```

Comment:
This should not be needed. Drop it?

We only need it in the other function to check if the changelog needs to be readded and new messages posted

Reply:
Ok. I just added it for consistency, but if we think no one will use it then I can remove.